cluster-state-service/handler/run/run.go

```diff
@@ -92,15 +95,28 @@ func StartClusterStateService(queueName string, bindAddr string, etcdEndpoints [
 	// start event processor
 	processor := event.NewProcessor(stores)
 
-	sqsClient := clients.NewSQSClient(awsSession)
 
-	// start event consumer
-	consumer, err := event.NewConsumer(sqsClient, processor, queueName)
-	if err != nil {
-		return errors.Wrapf(err, "Could not start the consumer")
-	}
+	if strings.HasPrefix(queueNameURI, kinesisPrefix) {
```

Comment:
This doesn't seem necessary.  The existing behavior seemed to validate sqs inside the sqs consumer constructor and to maintain backwards behavior, assuming an sqs queue seemed correct.  It seems that just looking for the kinesis prefix otherwise handing off to sqs is the desired behavior.